unix/main.c

```diff
@@ -416,7 +416,7 @@ int main(int argc, char **argv) {
             } else if (strcmp(argv[a], "-v") == 0) {
                 mp_verbose_flag++;
             } else if (strncmp(argv[a], "-O", 2) == 0) {
-                if (isdigit(argv[a][2])) {
+                if (isdigit((byte)argv[a][2])) {
```

Comment:
You should file a bug report to the standard comittee for defining isdigit as buggy :P